groupsforsections.py

```diff
@@ -255,7 +283,39 @@ def main():
 
         # mapping all the users in a sections to corresponding group
         groups_to_users_dict[group_id] = users
-        # TODO: Adding users to corresponding groups
+
+    failed_groups_to_users_dict={}
+    success_groups_to_users_dict={}
+
+    # adding users to the group
+    for group in groups_to_users_dict:
+        for user in groups_to_users_dict[group]:
```

Comment:
Rather than the nested `for` loops to iterate over `groups_to_users_dict`, use:

```python
    for group, user in groups_to_users_dict.iteritems():
```

It should be more efficient.